pkg/rtc/transport.go

```diff
@@ -326,6 +331,37 @@ func newPeerConnection(params TransportParams, onBandwidthEstimator func(estimat
 			ir.Add(f)
 		}
 	}
+
+	setTWCCForVideo := func(info *interceptor.StreamInfo) {
```

Comment:
:nit: just call this `setTWCC`. Although it is used only for `video`, does not need to be only video.

Reply:
Chrome's offer contains audio twcc fb and extension too and currently implementation is only enable twcc for video. I think enabling audio twcc is unrelate to RTX support, so keeps the behavior in this PR.